tools/mgba-rom-test-hydra/main.c

```diff
@@ -9,7 +9,11 @@
  * COMMANDS
  * N: Sets the test name to the remainder of the line.
  * R: Sets the result to the remainder of the line, and flushes any
- *    output buffered since the previous R. */
+ *    output buffered since the previous R.
+ *
+ * //Missing P, F and K documentation, please tell me what to put here lol
+ * 
+ */
```

Comment:
Oops, mb!

I suppose something like:
```
P/K/F: Sets the result to the remaining of the line, flushes any output since the previous P/K/F, and for P/K increments the number of passes/known fails.
```